src/cmd/sync.go

```diff
@@ -99,7 +102,7 @@ func executeSync(all, dryRun, verbose bool) error {
 			InitialBranch: data.initialBranch,
 			Remotes:       data.remotes,
 			Program:       &runProgram,
-			PushBranch:    true,
+			PushBranch:    !noPush,
```

Comment:
I believe this prevent _all_ branches in the sequence from being pushed? I'm thinking the behavior I want is that only the current feature branch is not pushed but could discuss the tradeoffs.

Reply:
I would certainly be surprised if I wrote `--no-push` and it pushed anything. I think preventing all branches from being pushed is the right move.

Reply:
I second that. If the stated goal is reducing CI cost/pressure, then it should not push any branches.

Reply:
That's a reasonable stance. Certainly fair to start with this behavior and see if it feels right over time.